src/inet/IPEndPointBasis.cpp

```diff
@@ -1154,7 +1154,7 @@ INET_ERROR IPEndPointBasis::ConfigureProtocol(IPAddressType aAddressType, nw_par
     return res;
 }
 
-INET_ERROR IPEndPointBasis::Bind(IPAddressType aAddressType, IPAddress aAddress, uint16_t aPort, nw_parameters_t aParameters)
+INET_ERROR IPEndPointBasis::Bind(IPAddressType aAddressType, IPAddress aAddress, uint16_t aPort, nw_parameters_t & aParameters)
```

Comment:
If we're toying with the API here, I'd also recommend making `IPAddress aAddress` `const IPAddress &aAddress` since I believe it to be read-only in this context and this will save space on both the caller side and on the stack for pushing the largish `IPAddress` structure on the stack rather than by reference (register width through a register).

Reply:
Done.